test/scraped/response/decorator/absolute_urls_test.rb

```diff
@@ -13,6 +13,7 @@
     <a class="external-link" href="http://example.org/person-123-other-page">Person 123</a>
     <a class="empty-link" href="">Person 123</a>
     <a class="javascript-link" href="javascript:chooseStyle('small',60);">Person 123</a>
+    <a class="relative-link-with-unencoded-space" href="/person 456">Person 456</a>
```

Comment:
All the other test examples use "Person 123". Switching to "Person 456" here is a little confusing, as it makes this test seem significantly different. If there _is_ some reason to make this different, I think it should be in a different test block, but I suspect just making consistent with the other tests would be OK.